x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/process/autodetect/writer/AbstractDataToProcessWriter.java

```diff
@@ -122,10 +122,10 @@ public void writeHeader() throws IOException {
 
     /**
      * Tokenize the field that has been configured for categorization, and store the resulting list of tokens in CSV
-     * format in the appropriate field of the record to be sent to the analytics.
+     * format in the appropriate field of the record to be sent to the dataframe.
```

Comment:
data frame

Reply:
This was meant to be the general meaning of `analytics`. I changed it to `process` to make it clearer.